spec/app/ics-028-cross-chain-validation/overview_and_basic_concepts.md

```diff
@@ -153,17 +153,21 @@ The channel initialization consists of four phases:
-  - *OnChanOpenTry*: On receiving the *FIRST* `ChanOpenTry` message, the provider CCV module sets the status of its end of the CCV channel to `INITIALIZING`.
-  - *OnChanOpenConfirm*: On receiving the *FIRST* `ChanOpenConfirm` message, the provider CCV module sets the status of its end of the CCV channel to `VALIDATING`.
-- **Channel completion**: Once the provider chain sets the status of the CCV channel to `VALIDATING`, 
-  it provides a VSC (i.e., validator set change) to the consumer chain (see [next section](#validator-set-update)). 
-  On receiving the *FIRST* `VSCPacket`, the consumer CCV module sets the status of its end of the CCV channel to `VALIDATING`. 
-
-> **Discussion**: As long as the [assumptions required by CCV](./system_model_and_properties.md#assumptions) hold (e.g., *Correct Relayer*), every governance proposal to spawn a new consumer chain that passes on the provider chain eventually results in a CCV channel being created. Furthermore, the "*FIRST*" keyword in the above description ensures the uniqueness of the CCV channel, i.e., all subsequent attempts to create another CCV channel to the same consumer chain will fail.
+  - *OnChanOpenInit*: On receiving a `ChanOpenInit` message, the consumer CCV module verifies that the underlying client associated with this channel is the expected client of the provider chain (i.e., created during genesis).
+  - *OnChanOpenTry*: On receiving a `ChanOpenTry` message, the provider CCV module verifies that the underlying client associated with this channel is the expected client of the consumer chain (i.e., created when handling the governance proposal).
+  - *OnChanOpenConfirm*: On receiving the *FIRST* `ChanOpenConfirm` message, the provider CCV module considers its side of the CCV channel to be established.
+- **Channel completion**: Once the provider chain side of the CCV channel is established, 
+  the provider CCV module provides VSCs (i.e., validator set changes) to the consumer chain (see the [next section](#validator-set-update)). 
+  On receiving the *FIRST* `VSCPacket`, the consumer CCV module considers its side of the CCV channel to be established. 
+  From this moment onwards, the consumer chain is secured by the provider chain.
+
+> **Note**: As long as the [assumptions required by CCV](./system_model_and_properties.md#assumptions) hold (e.g., *Correct Relayer*), every governance proposal to spawn a new consumer chain that passes on the provider chain results eventually in a CCV channel being created. 
+> Furthermore, the "*FIRST*" keyword in the above description ensures the uniqueness of the CCV channel, i.e., all subsequent attempts to create another CCV channel to the same consumer chain will fail.
+
+> **Note**: To reduce the attack surface during channel initialization, the consumer chain SHOULD enable user transactions only after the CCV channel is established (i.e., after receiving the first VSC). 
```

Comment:
Why is it "SHOULD"? Can we make explicit, what are the consequences of not following this advice.